yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -438,6 +519,15 @@ private[yarn] class YarnAllocator(
     amClient.releaseAssignedContainer(container.getId())
   }
 
+  private def rackLookUp(host: String, conf: Configuration): String = {
+    hostToRackNameCache.get(host) match {
```

Comment:
Thanks @sryza , I will remove this. Previously according to the comments of Marcelo I added it.